README.md

```diff
@@ -0,0 +1,29 @@
+# Team 3
+
+## Software Organization
+```
+core
+|__movement
+|  |__brushed_motor
+|  |__servo
+|__comms
+|  |__udp
+voice
+|__recognition
+|__command_parsing
+video
+|__processing
+|__streaming
+sensors
+|__hall
+|__imu
+|__range
+controller
+|__main_module.py
+|__steering_wheel
+|__hud
+wallu
+|__main_module.py
+|__wallu.ino
+schematics
```

Comment:
needs additional folders for different parts (LED eyes), Controller, WALL-U, etc.

Reply:
We can add those in later.